webidl/Generate.hx

```diff
@@ -399,7 +399,7 @@ template<typename T> pref<T> *_alloc_const( const T *value ) {
 			if( p.substr(0, 2) == "-O" )
 				hasOpt = true;
 		if( !hasOpt )
-			params.push("-O2");

```

Comment:
I think `O2` is probably a good default. I'm not really set on it if you have a good reason, but I think `O2` made sense.

Reply:
And that is also an oversight on my part (good catch).. I dropped down to -O0 so I could more easily read the output.. you are correct this should be optimised.